app/models/status.rb

```diff
@@ -31,6 +31,15 @@ class Status < ApplicationRecord
   include Cacheable
   include StatusThreadingConcern
 
+  LICENSE_URLS = {
+    'Attribution (CC BY)' => 'https://creativecommons.org/licenses/by/4.0/',
+    'Attribution-ShareAlike (CC BY-SA)' => 'https://creativecommons.org/licenses/by-sa/4.0/',
+    'Attribution-NoDerivs (CC BY-ND)' => 'https://creativecommons.org/licenses/by-nd/4.0/',
```

Comment:
The web page says "Attribution-NoDerivatives" (while it says "Attribution-NonCommercial-NoDerivs" for BY-NC-ND. How confusing!)